config/core/100-namespace.yaml

```diff
@@ -18,3 +18,5 @@ metadata:
   name: knative-serving
   labels:
     serving.knative.dev/release: devel
+    app.kubernetes.io/version: devel
+    app.kubernetes.io/part-of: knative
```

Comment:
Do you think `part-of` should be the subproject ie. `knative-serving` ?

Reply:
I can do that.